poi-examples/src/main/java/org/apache/poi/examples/xssf/streaming/SavePasswordProtectedXlsx.java

```diff
@@ -95,7 +96,7 @@ public static void save(final InputStream inputStream, final String filename, fi
             EncryptionInfo info = new EncryptionInfo(EncryptionMode.agile);
             Encryptor enc = Encryptor.getInstance(info);
             enc.confirmPassword(pwd);
-            opc.save(enc.getDataStream(fs));
+            opc.save(enc.getDataStream(fs, TempFileCreationStrategy.getDefaultStrategy()));
```

Comment:
please revert this change - there is no need to complicate the example with this

Reply:
reverted